specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -151,7 +151,7 @@ export function TreeRow({
             formatTreeStats(nodeStats, row.children === 0);
           return (
             <Button.LikeLink
-              aria-controls={id('children')}
+              aria-controls={displayChildren ? id('children') : undefined}
```

Comment:
from w3 and chatgpt it seems like `aria-controls` should not be set if the element it's pointing to does not exist (yet)